station/station_configs/testapp/docker-compose.yml

```diff
@@ -64,10 +64,10 @@ services:
     volumes:
      - "/etc/pulse:/etc/pulse:ro"
      - ${PULSE_SERVER}:/run/pulse/native
-     - ${PULSE_COOKIE}:/run/pulse/cookie
+#     - ${PULSE_COOKIE}:/run/pulse/cookie
```

Comment:
Not necessary anymore? Please explain.

Reply:
due to custom PulseAudio configuration for Supernova - there should be no cookie